backend/src/main/java/de/amos/apachepulsarui/service/NamespaceService.java

```diff
@@ -80,7 +80,13 @@ private NamespaceDetailDto enrichWithNamespaceData(NamespaceDetailDto namespace)
     }
 
     private NamespaceDto enrichWithCardDetails(NamespaceDto namespace) {
-            //TODO fill with information when given by pos
+        try {
+            namespace.setNumberOfTopics(pulsarAdmin.namespaces().getTopics(namespace.getId()).size());
```

Comment:
Same here. Can't we use the topic service for this?
`topicService.getAllByNamespace(topic).size();`